lib/lockService.js

```diff
@@ -71,7 +71,8 @@ module.exports = ({ sequelize, Sequelize }) => {
       .then(() => _createLock(name, timeoutInSec))
       .catch(err => {
         if (attempt < maxAttempt) {
-          logger.warn(`Cannot acquire the lock ${name}. Retry in ${delayInMs} milliseconds.`);
+          logger.warning(`Cannot acquire the lock ${name}. Retry in ${delayInMs} milliseconds.`);
+          logger.error('%s', err);
```

Comment:
I'm not sure we want to log an error here... you could at least move this statement a few lines below, just before`throw new Error(`Cannot acquire the lock '${name}'`);`